### PR TITLE
install python 3.7 for benchmark testing

### PR DESCRIPTION
**Pull request by carolynhu**
istio benchmark testing required python 3.7

### CODE REVIEW

**Comment by howardjohn** [comment]
Can we just not require python 3.7? That doesn't seem like a reasonable requirement considering gLinux doesn't even have 3.7

File: docker/istio/shared/tools/install-python.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 set -eux
 
+apt-get install python3.7
 apt-get -qqy install python3-pip
```
Comment (clarketm) [inline docker/istio/shared/tools/install-python.sh]: Doesn't this already install python 3?

**Comment by howardjohn** [comment]
I dont think we need this since we have https://github.com/istio/tools/pull/397

**Comment by carolynhu** [comment]
Yes, we do not need this. I'm going to close it.

**Pull request closed without merging**